### DISCUSSION TITLE
Summary story on attempting to install 6.0 20210615

### DISCUSSION THREAD

**Discussion by rsms** [elementary/installer#575] [General]
After having issues with Elementary 5, Today I wanted to try out installing 6.0 but after my budgeted time (1 hour) I had to give up.
Here is what I did and what happened

I downloaded elementaryos-6.0-daily.20210615.iso from https://builds.elementary.io and put it on a USB stick.
Booted up a PC with it. (ASRock X570 with a Ryzen 3950X)

The first issue I ran into was tiny UI which I couldn't make larger. (See #573)
I entered "demo" mode to try and find a way to fix the barely readable UI.
After finding the display settings and increasing the UI scale to 2.0 it was much easier to read the UI.

But now context menus were broken:

<img width="571" alt="ctxmenu" src="https://user-images.githubusercontent.com/47046/125173003-9ad2ea80-e171-11eb-9ca7-0241cd1fa453.png">

When looking at a slow-motion video, it appears as some portion of the desktop is sampled, scaled down and then animated to scale up (see how a portion of the desktop background image is "moving".) Perhaps that is a hint at what is going on.

![sdfg8n91o](https://user-images.githubusercontent.com/47046/125173051-d7064b00-e171-11eb-8411-1fcb443ba141.gif)

Next I struggled a little bit to find the installer. Eventually I tried tapping on the "applications" text in the top left corner of the screen and searching inside it for "install" and found the installer. I would expect the Elementary installer to be right on the desktop and/or in the "dock" at the bottom portion of the screen (by the way, being new to Elementary I have _no idea_ what those icons are at the bottom. Putting text titles there might be a good idea.)

After launching the installer and selecting a disk (a Samsung T5 1TB connected over USB-C) the installer was unable to find the disk and there were no tools available for poking around. In these two screen photos only the two internal disk are shown (also missing is the USB drive that I'm running Elementary "live" from.)

![smdpof1](https://user-images.githubusercontent.com/47046/125173168-c60a0980-e172-11eb-9790-a9a37414c247.png)

I had to cancel the installation, launch the demo "live" Elementary and used GParted to poke at the disk. It wasn't there. I plugged the Samsung T5 drive out of the machine and back in, waited, repeated that twice and suddenly it showed up. (I have had no issues with this drive on this same hardware when using Windows or Ubuntu, BTW.)

Restarting the installer this time the disk did show up and I choose the "easy" (suggested) path of erase & install. After a little while (~3s) the installer failed with this message: (I tapped the little "terminal" icon to show the actual error messages)

![IMG_0460](https://user-images.githubusercontent.com/47046/125173247-2ef18180-e173-11eb-813a-117b74ae9e35.jpeg)

[edit] Note: I did not choose disk encryption (I choose "no encryption")

So I cancelled the installer once again, launch the demo "live" Elementary and used GParted to wipe the disk, giving it a new partition table (GPT). Launched the installer again, selected the disk and this time it succeeded.

![IMG_0461](https://user-images.githubusercontent.com/47046/125173307-868fed00-e173-11eb-8e7c-dc7149b696d4.jpeg)

However, after the installation completed and I rebooted the computer, its EFI system was unable to find the disk (It found the other bootable disks, connected over USB and internal, but not the one Elementary just installed itself onto.) So, back to rebooting again and into the demo "live" Elementary from the USB stick. Opened GParted and looked at the drive (as it now appears after installing Elementary.) It looks fine — a single partition with the label "Elementary." I guesstimated that maybe my ASRock X570's EFI system has issues with GPT disks so I wiped the disk once again and used an msdos partition table.

Launched the installer, choose the disk and a few seconds in it failed with a to-me cryptic error:

![IMG_0462](https://user-images.githubusercontent.com/47046/125173399-0918ac80-e174-11eb-9db5-db794fcf435a.jpeg)

I was getting a little irritated at this point and just hit "retry" out of frustration. I really didn't think "trying the same thing again" would have different results but, lol, it did and the installer somehow worked this time. After it finished I rebooted my computer and again was disappointed to not find the disk showing up in the ASRock X570 EFI's "boot options" (again, internal disks and the USB stick with the installer did show up.)

![IMG_0467](https://user-images.githubusercontent.com/47046/125173541-c0152800-e174-11eb-84d4-a1be45373bbd.jpg)

At this point I cut my losses and gave up.
I understand Elementary 6.0 is beta software so I didn't have high expectations, but I thought it might be useful to share this story and I hope it can be helpful!

Keep up the good work!

**Comment by danirabbit** [comment]
Moved to a discussion since this is a kind of collection of various separate issues and not a single actionable issue. Makes it a bit easier to target to projects, assign, etc.

Thanks for writing this up! It sounds like a few obvious things we need to look into are:
- [ ] heuristics for GSD to switch to HiDPI
- [ ] external drives over USB C
- [ ] add the installer shortcut to the dock in the demo session
